src/bootstrap/dist.rs

```diff
@@ -422,33 +421,15 @@ impl Step for Rustc {
             let man_src = builder.src.join("src/doc/man");
             let man_dst = image.join("share/man/man1");
 
-            // Reproducible builds: If SOURCE_DATE_EPOCH is set, use that as the time.
-            let time = env::var("SOURCE_DATE_EPOCH")
```

Comment:
Isn't this reduce reproducible build quality?

Reply:
This was previously used for `<INSERT DATE HERE>` in the man pages. With this PR `<INSERT DATE HERE>` isn't used at all anymore.

Reply:
Ahh, yes, thanks.